app/controllers/alchemy/pictures_controller.rb

```diff
@@ -6,6 +6,8 @@ class PicturesController < Alchemy::BaseController
 
     before_filter :ensure_secure_params
 
+    helper_method :resource_url_proxy
```

Comment:
I think you don't need this helper. The `alchemy` url proxy should be available globally in your app. It's an engine.